### PR TITLE
apps create: just add a hint for quickstarts download

### LINKED ISSUE #127: Run quickstart after app creation [CLI-58]
Hint: download the quickstart `auht0 quickstarts download --clientId ....`

### PR DESCRIPTION
**Pull request by cyx**
Closes #127

### CODE REVIEW

File: internal/display/apps.go [hunk 1]
```diff
@@ -238,6 +238,10 @@ func (r *Renderer) ApplicationCreate(client *management.Client, revealSecrets bo
 		ansi.Faint("Hint:"),
 		client.GetClientID(),
 	)
+	r.Infof("%s You might wanna try 'auth0 quickstarts download --client-id %s'",
```
Comment (Widcket) [inline internal/display/apps.go:241]: Now it's `auth0 quickstarts download <client-id>` please check for other outdated instances.

**Review by Widcket — CHANGES_REQUESTED** [review]
The command has changed since.

File: internal/display/apps.go (hunk 1 above)
Comment (cyx) [reply internal/display/apps.go]: Couldn't find any other outdated refs for quickstarts download 👍

Comment (Widcket) [reply internal/display/apps.go]: Thanks!

**Review by Widcket — APPROVED** [review]
Nice!

**Pull request merged**